apps/src/netsim/NetSimRouterLogModal.js

```diff
@@ -130,6 +137,18 @@ var NetSimRouterLogModal = module.exports = function (rootDiv, options) {
    */
   this.teacherView_ = false;
 
+  /**
+   * Pre-bound callback for use when rendering.
+   * @private {function}
+   */
+  this.boundSetRouterLogMode_ = this.setRouterLogMode_.bind(this);
```

Comment:
I'm not sure. I know this is a common pattern in cases where you create React classes that extend React.Component. In that case, you don't get autobinding of methods and (at least one approach is to) often bind a bunch of local methods in the constructor.